src/mjbots_robot_interface.cpp

```diff
@@ -46,21 +46,20 @@ ::mjbots::moteus::QueryResult MjbotsRobotInterface::Get(const std::vector<::mjbo
   return {};
 }
 
-MjbotsRobotInterface::MjbotsRobotInterface(const std::vector<Motor> &motor_list,
+MjbotsRobotInterface::MjbotsRobotInterface(const std::vector<JointMoteus> &joint_list,
                                            const RealtimeParams &realtime_params,
-                                           float max_torque,
                                            int soft_start_duration) :
-    soft_start_(max_torque, soft_start_duration) {
-  // Process motor list
-  num_servos_ = motor_list.size();
-  for (const auto &motor_elem : motor_list) {
-    servo_id_list_.push_back(motor_elem.id);
-    servo_bus_list_.push_back(motor_elem.can_bus);
-    offsets_.push_back(motor_elem.offset);
-    directions_.push_back(motor_elem.direction);
+    soft_start_(100, soft_start_duration) { //TODO ADJUST ROBOT WIDE SOFTSTART TO DEFINE MAXTORQUE RAMP WITH INDIV JOINTS
```

Comment:
no. omg. why??? This is not something you should ever do. Don't disable a safety feature like this. Like if you need to change it make another constructor and use that for your tests. If I missed this and the code got merged in we could damage many things.